libshaderc_spvc/src/spvcir_pass.h

```diff
@@ -25,16 +25,31 @@ namespace opt {
 // this WIP pass generates spvc IR and does not throw exceptions
 class SpvcIrPass : public Pass {
  public:
-  SpvcIrPass(spirv_cross::ParsedIR& ir) { ir_ = ir; }
-  const char* name() const override { return "spvc-if-pass"; }
+  SpvcIrPass(spirv_cross::ParsedIR &ir);
+  const char *name() const override { return "spvc-if-pass"; }
   Status Process() override;
 
   IRContext::Analysis GetPreservedAnalyses() override {
     return IRContext::kAnalysisNone;
   }
 
+  void genSpirvCrossIR(Instruction *inst);
```

Comment:
Needs a method comment.
Style guide wants the method name to start with a capital.  (It's not a trivial accessor).